### PR TITLE
6572 clean up invalid QBO values for customer

### PR DESCRIPTION
**Pull request by X0nic**

### CODE REVIEW

**Comment by X0nic** [comment]
Add a `?w=1` to the URL to remove the whitespace changes.

https://github.com/sassafrastech/madeline/pull/337/files?w=1

File: app/models/accounting/quickbooks/customer.rb [hunk 1]
```diff
@@ -37,8 +36,10 @@ def service
       def find_or_create_qb_customer_id
         return organization.qb_id if organization.qb_id.present?
 
+        normalized_name = organization.name.tr(':', '_')
```
Comment (hooverlunch) [inline app/models/accounting/quickbooks/customer.rb]: Nice use of `tr`

File: spec/models/accounting/quickbooks/customer_spec.rb [hunk 2]
```diff
@@ -11,49 +10,66 @@
...
-      allow(service).to receive(:create).and_return(qb_new_customer)
+  context 'with invalid name' do
+    let(:customer_name) { 'A name:with a colon' }
+    let(:created_customer_name) { 'A name_with a colon' }
 
-      reference = subject.reference
-      expect(reference.type).to eq 'Customer'
-      expect(reference.entity_ref.value).to eq qb_customer_id
-    end
+    it_behaves_like 'qb customer'
```
Comment (hooverlunch) [inline spec/models/accounting/quickbooks/customer_spec.rb]: Nice job keeping these DRY!

**Comment by emitche** [comment]
The failed spec is not related to any of the changes made in this issue. I will place the need for a spec adjustment into the backlog.

**Pull request merged**